test/test_storage.py

```diff
@@ -218,6 +219,17 @@ class TestSQLiteStorage(StoragePermutations, BaseTestCase):
     def create_storage(self):
         return ops.storage.SQLiteStorage(':memory:')
 
+    def test_permissions(self):
+        fd, filename = tempfile.mkstemp()
```

Comment:
As discussed, let's use https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory with a `with statement` for this.